UOP1_Project/Assets/Scripts/CameraSwapTrigger.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Cinemachine;
+
+public class CameraSwapTrigger : MonoBehaviour
+{
+    public CinemachineVirtualCamera cameraToSwap;
+    
+    [Header("Asset references")]
+    [SerializeField] private VcamEventChannelSO VcamEventChannel = default;
+
+    BoxCollider boxCollider;
```

Comment:
Might be better not to assume it's a Box, use generic `Collider` class instead?

Reply:
implemented.